app/controllers/api/conferences/groups_controller.rb

```diff
@@ -10,11 +10,12 @@ def create
   end
 
   def index
+    groups = Group.all
     render json: Group.all, each_serializer: GroupIndexSerializer
   end
 
   def show
-    group = Group.find params[:id]
+    group = Group.includes(:conference, students: :school).find params[:id]
```

Comment:
Since we know that the `GroupShowSerializer` (in the line below) will be including a `conference` model and `student` models, we include those in the query for the group. This saves extra queries by the serialization system, since our `group` instance comes with all its nested instances. `students: :school` is a deep nest, which means we include the `students` of a group plus the `school` of each of the `students` (can you deduce why this is necessary?).